packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/dto/PageInfoWithCount.java

```diff
@@ -10,7 +10,7 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-package org.eclipse.sirius.web.application.dto;
+package org.eclipse.sirius.components.collaborative.dto;
```

Comment:
Why not in `org.eclipse.sirius.components.core.graphql.dto`? This does not seem to be used in `sirius-components-collaborative` and thus the dependency to `graphql-java` seems useless here.

Reply:
Moved in `org.eclipse.sirius.components.core.graphql.dto`